content/en/checkout/reference.md

```diff
@@ -107,18 +102,22 @@ An object with the following properties:
 
 - `manifest`: The return object from the [checkout service](#checkout).
 - `options`: An optional object containing any of the following properties:
-  | Option | Type | Required | Default | Description |
-  | ------------ | ---------- | -------- | ------------------ | -------------------------------------------------------------------------- |
-  | `id` | `string` | no | `easypay-checkout` | The id of the HTML element where the Checkout form should be included. |
-  | `onSuccess` | `function` | no | `() => {}` | Callback function to be called when the Checkout is finished successfully. |
-  | `onError` | `function` | no | `() => {}` | Callback function to be called on errors. |
-  | `onClose` | `function` | no | `undefined` | Callback function to be called when the Checkout popup is closed. |
-  | `testing` | `boolean` | no | `false` | Whether to use the testing API (`true`) or the production one (`false`). |
-  | `display`(1) | `string` | no | `inline` | The display style of the element that hosts the Checkout |
+  | Option        | Type       | Required | Default            | Description                                                                |
+  | ------------- | ---------- | -------- | ------------------ | -------------------------------------------------------------------------- |
+  | `id`          | `string`   | no       | `easypay-checkout` | The id of the HTML element where the Checkout form should be included.     |
+  | `onSuccess`   | `function` | no       | `() => {}`         | Callback function to be called when the Checkout is finished successfully. |
+  | `onError`     | `function` | no       | `() => {}`         | Callback function to be called on errors.                                  |
+  | `onClose`     | `function` | no       | `undefined`        | Callback function to be called when the Checkout popup is closed.          |
+  | `testing`     | `boolean`  | no       | `false`            | Whether to use the testing API (`true`) or the production one (`false`).   |
+  | `display`(1)  | `string`   | no       | `inline`           | The display style of the element that hosts the Checkout.                  |
+  | `hideDetails` | `boolean`  | no       | `false`            | Whether to hide the details form or not. A summary will be shown with the customer information initially sent in the Checkout start service call. |
```

Comment:
I think the last sentence `A summary will be...` should be updated for something lighter like `A expandable box will be displayed with the customer details, instead`